CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Are we sure this is fine to bump?

Reply:
It is the minimal CMake version in the CI builds. Since that is the only one we can support, I think this is fine to bump.

Reply:
Ok